frontend/amp/pages/MainBlock.tsx

```diff
@@ -0,0 +1,273 @@
+    }
+    padding: 0 10px;
+`;
+
+const header = css`
+    margin: 0 -10px;
+`;
+
+const section = css`
+    @supports (display: grid) {
+        grid-template-areas: 'section';
+    }
+    font-size: 16px;
+    line-height: 20px;
+    font-family: ${serif.headline};
+    font-weight: 700;
+
+    padding: 0 10px;
+`;
+
+const profile = css`
+    font-size: 16px;
+    line-height: 20px;
+    font-family: ${serif.headline};
+    font-weight: 700;
+    margin-bottom: 4px;
+`;
+
+const sectionLabelLink = css`
+    text-decoration: none;
+    :hover {
+        text-decoration: underline;
+    }
+`;
+
+const listStyles = css`
+    li {
+        font-family: ${serif.body};
+        margin-bottom: 6px;
+        padding-left: 20px;
+        font-size: 17px;
+        line-height: 24px;
+
+        p {
+            display: inline;
+        }
+    }
+
+    li:before {
+        display: inline-block;
+        content: '';
+        border-radius: 6px;
+        height: 12px;
+        width: 12px;
+        margin-right: 8px;
+        background-color: ${palette.neutral[86]};
+        margin-left: -20px;
+    }
+`;
+
+const standfirst = css`
+    font-family: ${serif.body};
+    font-weight: 700;
+    font-size: 17px;
+    line-height: 22px;
+    color: ${palette.neutral[7]};
+    margin-bottom: 12px;
+
+    ${listStyles};
+
+    p {
+        margin-bottom: 8px;
+    }
+`;
+
+const ageWarning = css`
+    font-size: 12px;
+    line-height: 16px;
+    font-family: ${sans.body};
+    display: inline-block;
+    margin-bottom: 12px;
+    width: 100%;
+`;
+
+const standfirstLinks = pillarMap(
+    pillar =>
+        css`
+            a {
+                color: ${pillarPalette[pillar].dark};
+                text-decoration: none;
+                border-bottom: 1px solid ${palette.neutral[86]};
+                transition: border-color 0.15s ease-out;
+            }
+        `,
+);
+
+const pillarColours = pillarMap(
+    pillar =>
+        css`
+            color: ${pillarPalette[pillar].main};
+        `,
+);
+const pillarFill = pillarMap(
+    pillar =>
+        css`
+            fill: ${pillarPalette[pillar].main};
+        `,
+);
+
+const twitterHandle = css`
+    font-size: 12px;
+    line-height: 16px;
+    font-family: ${sans.body};
+    font-weight: bold;
+    color: ${palette.neutral[46]};
+
+    padding-right: 10px;
+    display: inline-block;
+
+    svg {
+        height: 10px;
+        max-width: 12px;
+        margin-right: 0px;
+        fill: ${palette.neutral[46]};
+    }
+
+    a {
+        color: ${palette.neutral[46]};
+        text-decoration: none;
+    }
+`;
+
+const metaExtras = css`
+    border-top: 1px solid ${palette.neutral[86]};
+    padding-top: 6px;
+    margin-bottom: 6px;
+    display: flex;
+    justify-content: space-between;
+    flex-wrap: wrap;
+
+    margin-left: -10px;
+    margin-right: -10px;
+    padding-left: 10px;
+    padding-right: 10px;
+`;
+
+export const MainBlock: React.SFC<{
+    CAPI: CAPIType;
```

Comment:
Is it possible to pass in just what is needed? As CAPI is a pretty big interface to require here. (The cons are that it doesn't document what is actually needed by the component very well, and also makes it more difficult for the calling code, including tests, as it has to construct a complex CAPI object.)

One way to make it easier would be to still pass the CAPI object (as that already exists), but have the argument type be just the subset required (an interface defined in this file). This would make it easier to change upstream behaviour in future or add tests.